packages/react/package.json

```diff
@@ -33,8 +33,8 @@
 	"devDependencies": {
 		"@testing-library/jest-dom": "^6.4.2",
 		"@testing-library/react": "^14.3.1",
-		"@types/react": "^18.2.79",
-		"@types/react-dom": "^18.2.25",
+		"@types/react": "^19.0.10",
+		"@types/react-dom": "^19.0.4",
```

Comment:
I'd propose to do it like they do in radix-ui and keep the package react and @types/react version in sync (v18) and instead declare an optional peer dependency, see https://github.com/radix-ui/primitives/blob/main/packages/react/accordion/package.json#L52-L65

That should work in the cases of react v18 and v19 then. 

Can you give that a try? Thank you!